.changeset/few-forks-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/sheet': patch
```

Comment:
Should be a major

Reply:
Technically in a regular situation - yes, this kind of a change should be a major. Ive deliberetely made it patch because this method was only added within current prerelease cycle so technically this is not a breaking change for v10 users. 

I will be removing this changeset and changing the original one before landing v11 so no mention of rehydrate will be included in the final v11 changelogs.

Also - as I plan to remove this particular changeset i didnt want to trigger too high bump in the current prerelease cycle as removing it later would cause a final version to be downgraded (in comparison to one of released prereleases) and this would only cause confusion and potentially be a problem later.